packages/swapper/src/manager/SwapperManager.test.ts

```diff
@@ -59,11 +60,11 @@ describe('SwapperManager', () => {
     })
   })
 
-  describe('getSwapper', () => {
+  describe('get', () => {
```

Comment:
I'd probably call this one `can get swapper instance from swappers` or something descriptive like so

Reply:
Right you are, that was a lazy test name. Updated in https://github.com/shapeshift/lib/pull/853/commits/0baf714b97e3ab6b870a46bf2e3913cba454e518.